app/konnect/api-products/service-documentation.md

```diff
@@ -24,9 +18,25 @@ You can provide extended descriptions of your API products with a Markdown (`.md
   <img src="/assets/images/products/konnect/api-products/konnect_service_docs_description.png" />
 </p>
 
-All API product descriptions are managed from the **Documentation** section in the API Product overview. Once you've uploaded the markdown file, you have a preview of how it will render, the option to edit, and a view of the publication status. You can also create a hierarchy between the docs you upload that will be reflected in the way they're displayed in the Dev Portal.
 
-## API specification
+Manage all API product descriptions from the Documentation section in the API Product overview. After uploading your markdown file, you'll be able to preview its rendering, edit it directly using the {{site.konnect_saas}} Markdown Renderer, and check its publication status. Additionally, you can organize your documents into a hierarchy, which will influence the order they are displayed to your users from within the Dev Portal.
+
+### Interactive markdown renderer
+
+The integrated markdown editor allows you to edit your documentation directly within {{site.konnect_saas}}. It supports:
+
+* Code syntax highlighting for Bash, JSON, Go, and JavaScript
+* Support for rendering UML diagrams and flowcharts via Mermaid and PlantUML
+* Support for emojis
```

Comment:
```suggestion
* Rendering UML diagrams and flowcharts via Mermaid and PlantUML
* Emojis
```
Just removed the "support" part to avoid repetition because it would be "It supports: support for rendering..."